twd.htm

```diff
@@ -207195,18 +207142,21 @@ <h3><a id="Year1997">1997</a></h3>
 Archon Investigation 4
 Blood Doll 4
 
-PRESENCE 40
 Majesty 8
 Bewitching Oration 5
 Awe 8
 Charming Loby 8
 Voter captivation 8
 Entrancement 3
 
-VOTES 28
 Kine resources Contested 5
 Conservative agitation 3
-Praxis Seizure ... 6
+Praxis Seizure: Athens 1
+Praxis Seizure: Atlanta 1
+Praxis Seizure: Amsterdam 1
+Praxis Seizure: Barcelona 1
+Praxis Seizure: Berlin 1
+Praxis Seizure: Boston 1
```

Comment:
a) This change should be noted 
b) I'd avoid included cards like PS: Berlin which have a clan benefit, and just use purely neutral versions.

Reply:
PS: Barcelona has a Tremere clause - I'd use Dallas instead.

Reply:
Done